### PR TITLE
TG Update

### PR DESCRIPTION
**Pull request by TheSwain**
Core update

### CODE REVIEW

**Comment by deathride58** [comment]
Wrong repo. Lol.

**Comment by Qustinnus** [comment]
lmao he pulled  a j_mad

**Comment by Incoming5643** [comment]
Good to see yall are still going

**Comment by AuroranAI** [comment]
https://youtu.be/1SHK4J392rI?t=12s

File: code/controllers/configuration/entries/config.dm [hunk 1]
```diff
@@ -171,6 +171,9 @@ CONFIG_DEF(number/githubrepoid)
 	value = null
 	min_val = 0
 
+CONFIG_DEF(string/discordurl)
+	value = "https://discord.gg/SVu782A"
```
Comment (optimumtact) [inline code/controllers/configuration/entries/config.dm]: nwah

File: code/__DEFINES/bloodsucker_defines.dm [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+// These are placed here because in some cases, if they are placed in the Bloodsucker antag folder, they won't get read by mind.dm, etc.
+
+#define ANTAG_DATUM_BLOODSUCKER			/datum/antagonist/bloodsucker
```
Comment (Qustinnus) [inline code/__DEFINES/bloodsucker_defines.dm]: succ

File: strings/station_names.txt [hunk 3]
```diff
@@ -1,82 +1,263 @@
...
+Neal
+Manuel
+Landon
+Keegan
+Butch
+Buck
+Kane
+Jasper
+Smalls
+Ponyboy
```
Comment (optimumtact) [inline strings/station_names.txt]: Bylat

File: strings/round_start_sounds.txt [hunk 4]
```diff
@@ -1,4 +1,3 @@
 sound/ambience/title1.ogg
 sound/ambience/title2.ogg
 sound/ambience/title3.ogg
-sound/ambience/clown.ogg
```
Comment (Qustinnus) [inline strings/round_start_sounds.txt]: codebase ruined

File: config/admins.txt [hunk 5]
```diff
@@ -7,127 +7,36 @@
...
+saladata	=	Administrator
+RootVeggie		=	Administrator
+Hugh Mann		=	Administrator
+Krzychukoko		=	Administrator
+Hestymk1		=	Administrator
+Patrick10ghost		=	Administrator
+DrachenKlaue		=	Administrator
+ChocolateJ		=	Administrator
+Axolotl123		=	Administrator
+Qustinnus		=	Administrator
```
Comment (Qustinnus) [inline config/admins.txt]: dead

File: config/admins.txt [hunk 6]
```diff
@@ -7,127 +7,36 @@
...
+Idontcareacount		=	Administrator
+
+#TRIAL ADMINS
+blank		=	Trial Admin
+Ss4gogeta0	=	Trial Admin
```
Comment (Qustinnus) [inline config/admins.txt]: hahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahahav

**Comment by Qustinnus** [comment]
image
lmao im taking this sprite

**Comment by TheSwain** [comment]
Somehow THIS KEEPS HAPPENING TO ME.

One day I'll learn @)(*$@)_(% Github.

Also, I'm counting the days til I can start contributing this goddamn Bloodsucker antag mode to TG. It needs HARDCORE TESTING.

**Comment by TheSwain** [comment]
Anyway, see you guys next Monday when I inevitably pull Fulpstation to TGstation again, TATA

**Comment by deathride58** [comment]
Just use the PR mirror script that FTL13, Hippie, and Citadel use. It's far more robust than manually merging from upstream every single week.

File: strings/round_start_sounds.txt (hunk 4 above)
Comment (deranging) [reply strings/round_start_sounds.txt]: i agree

**Pull request closed without merging**